tests/integration/conftest.py

```diff
@@ -669,7 +732,7 @@ def assert_exists(self):
             AssertionError: Test case does not exist.
         """
 
-        query = "'name' ~ '{}'".format(self.name)
+        query = "'Automation Content' ~ '{}'".format(self.test_id)
```

Comment:
This was a subtle bug. When searching for test cases normally searching by name is fine. However, given that test cases can be renamed it is better to look test cases up by the `test_id` instead.